docs/api-reference.md

```diff
@@ -509,6 +510,21 @@ onTakeSnapshot((copyOfLayout) => {
 });
 ```
 
+### function: onContextMenu(addItemCallback)
+
+- `addItemCallback` <[Function]>
+
+Registers a callback that is called when a a cell value is copied.
+
+```js
+import { onContextMenu } from '@nebula.js/stardust';
+
+onContextMenu((menu, event) => {
+  menu.addItem(item, index);
+  return Promise.resolve(menu);
+});
+```
```

Comment:
Should I have this code block here ?

Reply:
Nope

Reply:
Ok I'll remove it. btw, the similar hook for snapshot is here, shouldn't the onContextMenu be as well ?